Examples/MAX32520/UCL/project.mk

```diff
@@ -0,0 +1,24 @@
+# This file can be used to set build configuration
+# variables.  These variables are defined in a file called 
+# "Makefile" that is located next to this one.
+
+# For instructions on how to use this system, see
+# https://github.com/Analog-Devices-MSDK/VSCode-Maxim/tree/develop#build-configuration
+
+#BOARD=MAX32520FTHR
+# ^ For example, you can uncomment this line to make the 
+# project build for the "MAX32520FTHR" board.
+
+# **********************************************************
+
+# Enable UCL library
+LIB_UCL = 1
+export UCL_VERSION=2.7.0
+
+# Add project's include and source paths
+VPATH += ./src
```

Comment:
All UCL projects in the PR are missing the source code, is this intentional because of NDA rules?

Reply:
UCL team has different work flow, example source code is used for all SDKs (MML, LP, MSDK).
It is not a good idea to keep a copy of files in here.
My purpose by this commit only centralize project files (.eclipse, makefiles...) not source code.

Reply:
Ok, where is the source code hosted?

Reply:
It is in Gerrit.